pygluu/kubernetes/gui/forms.py

```diff
@@ -0,0 +1,783 @@
+        default="gluu-gateway",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    gluu_gateway_ui_namespace = StringField(
+        "Please enter a namespace for gluu gateway ui",
+        default="gg-ui",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    kong_database = StringField(
+        "Please enter gluu-gateway postgres database name",
+        default="kong",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    kong_pg_user = StringField(
+        "Please enter a user for gluu-gateway postgres database",
+        default="konga",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    kong_pg_password = StringField(
+        "Kong Postgress Password",
+        widget=PasswordInput(hide_value=False),
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y"),
+                    password_requirement_check],
+        description="Password is randomly generated with 6 characters contain "
+                    "number, uppercase letter, lower case letter and symbol")
+    kong_pg_password_confirm = StringField(
+        "Kong Postgress Password Confirmation",
+        widget=PasswordInput(hide_value=False),
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y"),
+                    EqualTo('kong_pg_password_confirm',
+                            message='Passwords do not match')])
+    gluu_gateway_ui_database = StringField(
+        "Please enter gluu-gateway-ui postgres database name",
+        default="kong",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    gluu_gateway_ui_pg_user = StringField(
+        "Please enter a user for gluu-gateway-ui postgres database",
+        default="konga",
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y")])
+    gluu_gateway_ui_pg_password = StringField(
+        "Gluu Gateway UI postgres password",
+        widget=PasswordInput(hide_value=False),
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y"),
+                    password_requirement_check],
+        description="Password is randomly generated with 6 characters contain "
+                    "number, uppercase letter, lower case letter and symbol")
+    gluu_gateway_ui_pg_password_confirm = StringField(
+        "Gluu Gateway UI postgres password confirmation",
+        widget=PasswordInput(hide_value=False),
+        validators=[RequiredIfFieldEqualTo("install_gluu_gateway", "Y"),
+                    EqualTo('gluu_gateway_ui_pg_password',
+                            message='Passwords do not match')])
+
+
+class JackrabbitForm(FlaskForm):
+    install_jackrabbit = RadioField("Install Jackrabbit",
+                                    choices=[("Y", "Yes"), ("N", "No")],
+                                    default="Y",
+                                    validators=[DataRequired()])
```

Comment:
Add Jackrabbit cluster mode field here. Check latest prompt.py for the list of settings. Cluster mode automatically enables installation of Postgres.